.changeset/gold-games-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-docs/gatsby-theme-docs': patch
+---
+
+Set spacing between description and API type in wide layout
```

Comment:
```suggestion
Do not remove top margin for any side by side content.  Resolves the API type having been too near to the description and manual side by side content being too near to the heading above.
``

Reply:
Makes sense...Thanks for the suggestion!  I will remember that 👍